oauth2_http/javatests/com/google/auth/oauth2/DefaultCredentialsProviderTest.java

```diff
@@ -71,6 +75,10 @@
 
   private static final String USER_CLIENT_SECRET = "jakuaL9YyieakhECKL2SwZcu";
   private static final String USER_CLIENT_ID = "ya29.1.AADtN_UtlxN3PuGAxrN2XQnZTVRvDyVWnYq4I6dws";
+  private static final String GCLOUDSDK_CLIENT_ID =
+      "764086051850-6qr4p6gpi6hn506pt8ejuq83di341hur.apps.googleusercontent.com";
+  private static final String GCLOUD_WARNING_CONTENTS =
```

Comment:
Since this is only used once, and is not really generic, I'm in favour of not having a variable for it. Thoughts?

Reply:
SGTM, moved to an inline string.